src/cnc/flavors/aws/ecs/1/provision/base.tf.j2

```diff
@@ -16,6 +16,28 @@ terraform {
 
 {% block collection_level_resources %}
 {% include "partials/collection_level_resources.tf.j2" %}
+
+{% if env_collection.serverless_services %}
+{% block lambda %}
+{% include "partials/collection/lambda/lambda.tf.j2" %}
```

Comment:
is the "lambda_function" partial needed as well? I see both here and look similar, not sure the difference

Reply:
It is necessary until we move the replicated code to shared